kolibri/plugins/learn/assets/src/learn-main.vue

```diff
@@ -21,4 +22,12 @@
 </script>
 
 
-<style lang="stylus" scoped></style>
+<style lang="stylus" scoped>
+
+  @require '~theme.styl'
```

Comment:
So even though `@require 'theme.styl'` appears in core-base, I must still require it here?